### ISSUE TITLE
HTML Footnotes appearing only in references page

### ISSUE THREAD

**Issue by Fed03** [rstudio/bookdown#1293]
Hi,
It seems that the bug mentioned in https://github.com/rstudio/bookdown/issues/793 made its return.
Tried on a fresh version of bookdown-demo. If you navigate to the Methods page, you'll se 2 footnotes are defined but none get displayed at the bottom of page

<img width="1536" alt="Schermata 2021-12-24 alle 18 25 58 (2)" src="https://user-images.githubusercontent.com/1193756/147366931-ca33086b-e76e-4ff6-88a2-0b0c58c2f1d3.png">

This is some info
```
R version 4.1.2 (2021-11-01)
Platform: x86_64-apple-darwin21.1.0 (64-bit)
Running under: macOS Big Sur 10.16

Locale: en_US.UTF-8 / en_US.UTF-8 / en_US.UTF-8 / C / en_US.UTF-8 / en_US.UTF-8

Package version:
  base64enc_0.1.3 bookdown_0.24   digest_0.6.29   evaluate_0.14   fastmap_1.1.0  
  glue_1.6.0      graphics_4.1.2  grDevices_4.1.2 highr_0.9       htmltools_0.5.2
  jquerylib_0.1.4 jsonlite_1.7.2  knitr_1.36      magrittr_2.0.1  methods_4.1.2  
  rlang_0.4.12    rmarkdown_2.11  stats_4.1.2     stringi_1.7.6   stringr_1.4.0  
  tinytex_0.35    tools_4.1.2     utils_4.1.2     xfun_0.28       yaml_2.2.1  
```
Installed pandoc version is 2.16.2

Thanks
CC @cderv

**Comment by cderv** [comment]
I think this is an issue fixed in current dev version (next 0.25 version): see [NEWS](https://github.com/rstudio/bookdown/blob/main/NEWS.md#changes-in-bookdown-version-025)

It was introduced by a change in Pandoc 2.15 - see https://github.com/rstudio/bookdown/pull/1275

Please update your bookdown version to use the dev version. 

```r
remotes::install_github("rstudio/bookdown")
```

Thanks!

**Comment by Fed03** [comment]
Cool, it's fixed indeed! many thanks 🙏
